packages/stitch/src/subschemaConfigTransforms/splitMergedTypeAccessTransformer.ts

```diff
@@ -0,0 +1,51 @@
+import { cloneSubschemaConfig, SubschemaConfig } from '@graphql-tools/delegate';
+
+export function splitMergedTypeAccessTransformer(subschemaConfig: SubschemaConfig): Array<SubschemaConfig> {
+  if (!subschemaConfig.merge) return [subschemaConfig];
+
+  const maxAccessors = Object.values(subschemaConfig.merge).reduce((max: number, mergedTypeConfig) => {
+    return Math.max(max, mergedTypeConfig?.accessors?.length ?? 0);
+  }, 0);
+
+  if (maxAccessors === 0) return [subschemaConfig];
+
+  const subschemaPermutations = [];
+
+  for (let i = 0; i < maxAccessors; i += 1) {
+    const subschemaPermutation = cloneSubschemaConfig(subschemaConfig);
+    const mergedTypesCopy = subschemaPermutation.merge;
+
+    if (i > 0) {
+      subschemaPermutation.merge = Object.create(null);
```

Comment:
This errs on putting the _least_ amount of merged type config into later subschemas. Alternatively, would it be better to include _as much_ merged type config as possible into each subschema to maximize local access?

Reply:
Maximize better if I understand correctly

Reply:
Actually, I think minimum fine as long as batching is enabled

Reply:
Going to leave this as minimum then, because maximizing won't necessarily include everything when multiple types have multiple entry points.